src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/db/RecallDocument.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.managerecallsapi.db
+
+import java.util.UUID
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table(name = "recall_document")
+data class RecallDocument(
+  @Id
+  val id: UUID,
+
+  @Column(name = "recall_id", nullable = false)
+  val recallId: UUID,
+
+  @Column(name = "category", nullable = false)
+  @Enumerated(EnumType.STRING)
+  val category: RecallDocumentCategory
```

Comment:
I think we said we were going to store file name too?

Reply:
I was thinking the file name could be `{category}.pdf` since there's only one category of document per recall - if necessary, we can always add it later.